riscv/tests/instructions.rs

```diff
@@ -1,17 +1,29 @@
 mod common;
 
 mod instruction_tests {
-    use crate::common::verify_riscv_asm_string;
+    use std::path::Path;
+
+    use crate::common::{verify_riscv_asm_file, verify_riscv_asm_string};
     use powdr_backend::BackendType;
     use powdr_number::GoldilocksField;
     use powdr_riscv::asm::compile;
     use powdr_riscv::Runtime;
     use test_log::test;
 
-    fn run_instruction_test(assembly: &str, name: &str) {
+    fn run_instruction_test(path: &Path, name: &str) {
+        // Test from ELF path:
+        verify_riscv_asm_file(path, &Runtime::base(), false);
```

Comment:
nitpick as this is only used here i think: have `elf` in the function name somehow?

Reply:
But this also executes the test from the assembly path.

Reply:
ah right, i got confused by the `asm` in the name, but asm here is from the input not output